### ISSUE TITLE
Please add a manual reload button on sidebar

### ISSUE THREAD

**Issue by WSQsGithub** [mcndt/obsidian-toggl-integration#139]
I am often unable to stop a timer so I have to close obsidian and reopen it. It would be much convenient if you add a reload button on the page to manually sync the timer. Thank you very much.

**Comment by 76563** [comment]
Me too, my toggl is often unresponsive since the update, so that I need to reopen obsidian

**Comment by AskPascal** [comment]
Same here. Great to see that you are working on a (partial) fix @leoccyao

**Comment by ushham** [comment]
Yep same here. It always breaks when I loose wifi for a moment or switch on my VPN. Closing and reopening obsidian is slow and a manual reload button would fix all these issues.

**Comment by Benature** [comment]
same issue

**Comment by AskPascal** [comment]
I currently work around this by deactivating and activating the plugin from the settings. But still an automatic reconnect or at least a button in the pane would be great!

**Comment by Benature** [comment]
For now, similar to @AskPascal , I install https://github.com/Benature/obsidian-plugin-reloader by BRAT as a workaround -- reload the toggl track plugin on the command palette. The plugin is only to simplify the reload process, and I can bind the command to a hotkey or ribbon.

![image](https://github.com/mcndt/obsidian-toggl-integration/assets/35028647/aa1e96bf-a394-4894-99d2-9e6570a55e47)

**Comment by ushham** [comment]
Great tip, that's a great temporary work around, thank you!
